package.json

```diff
@@ -64,6 +64,8 @@
     "@babel/core": "^7.16.0",
     "@babel/eslint-parser": "^7.16.3",
     "@semantic-release/git": "^10.0.1",
+    "appium": "^2.0.0-beta.40",
+    "appium-mac2-driver": "file:.",
```

Comment:
what's this for?

Reply:
to be able to start a local server for e2e tests, a hint from @jlipps : https://appium.slack.com/archives/C02956V3H/p1655144800931379